renderers/photoaudiohaptics.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/renderers/photoaudiohaptics.schema.json",
+    "title": "Photo Audio Haptic Renderer",
+    "type": "object",
+    "description": "Data for basic audio haptics selection renderer ca.mcgill.a11y.image.renderer.PhotoAudioHaptics.",
+    "definitions": {
+        "centroid": {
+            "type": "array",
+            "description": "The normalized geometric center of the entity.",
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+            },
+            "minItems": 2,
+            "maxItems": 2
+        },
+        "coordinates": {
+            "type": "array",
+            "description": "The normalized coordinates for each entity.",
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+            }
+        },
+        "contourPoints": {
+            "type": "array",
+            "description": "An array containing an array of normalized coordinates representing a point on the segment or point on the bounding box.",
+            "items": {
+                "$ref": "#/definitions/coordinates"
+            }
+        }
+    },
+    "properties": {
+        "image": {
+            "type": "string",
+            "description": "A base64 encoding of the image as a string.",
+            "pattern": "data:image\/[^;]+;base64[^\"]+"
+        },
+        "info": {
+            "type": "object",
+            "properties": {
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "audioFile": {
+                            "$ref": "./definitions.json#/definitions/audioFile"
+                        },
+                        "entityInfo": {
+                            "type": "array",
+                            "description":  "An entity contains the relevant audio-haptic information to locate each segment or object.",
```

Comment:
To clarify, an entity is data about an object or segment? How is this ordered?

Reply:
Segments, followed by objects. Could be clarified in the description.